src/test/java/io/vertx/ext/asyncsql/PostgreSQLTest.java

```diff
@@ -78,4 +78,53 @@ public void someTest() throws Exception {
     await();
   }
 
+  @Test
+  public void queryTypeTimestampWithTimezoneTest() throws Exception {
+    asyncSqlClient.getConnection(onSuccess(conn -> {
+      conn.execute("CREATE TABLE IF NOT EXISTS timestamptest (ts timestamp with time zone)", onSuccess(resultSet -> {
+        conn.execute("INSERT INTO timestamptest (ts) VALUES (now())", onSuccess(rs1 -> {
+          conn.query("SELECT * FROM timestamptest;", onSuccess(rs2 -> {
+            assertNotNull(rs2);
+            assertNotNull(rs2.getResults());
```

Comment:
It would be good to add at least some check that the data received is a correctly formed date or something like that.